src/index.ts

```diff
@@ -1,4 +1,5 @@
-import { pipe, pipeWith } from 'pipe-ts';
```

Comment:
Since we had to use `fp-ts` in order to use `monocle-ts`, I migrated the pipe functions from `pipe-ts` to `fp-ts`.